src/main/java/no/nav/foreldrepenger/oversikt/oppgave/MinSideBeskjedMorsArbeidTask.java

```diff
@@ -0,0 +1,80 @@
+package no.nav.foreldrepenger.oversikt.oppgave;
+
+
+import static no.nav.foreldrepenger.oversikt.oppgave.MinSideBeskjedMorsArbeidTask.TASK_TYPE;
+
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+import java.util.UUID;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import no.nav.foreldrepenger.oversikt.domene.SakRepository;
+import no.nav.foreldrepenger.oversikt.domene.Saksnummer;
+import no.nav.foreldrepenger.oversikt.domene.YtelseType;
+import no.nav.foreldrepenger.oversikt.domene.fp.ForeldrepengerSak;
+import no.nav.vedtak.felles.prosesstask.api.ProsessTask;
+import no.nav.vedtak.felles.prosesstask.api.ProsessTaskData;
+import no.nav.vedtak.felles.prosesstask.api.ProsessTaskHandler;
+
+@ApplicationScoped
+@ProsessTask(value = TASK_TYPE, prioritet = 3)
+public class MinSideBeskjedMorsArbeidTask implements ProsessTaskHandler {
+
+    public static final String TASK_TYPE = "dittnav.beskjed.morsarbeid";
+    public static final Duration DELAY_MIN = Duration.ofMinutes(15);
+
+    private static final Logger LOG = LoggerFactory.getLogger(MinSideBeskjedMorsArbeidTask.class);
+
+    private MinSideTjeneste minSideTjeneste;
+    private SakRepository sakRepository;
+
+    @Inject
+    MinSideBeskjedMorsArbeidTask(MinSideTjeneste minSideTjeneste, SakRepository sakRepository) {
+        this.minSideTjeneste = minSideTjeneste;
+        this.sakRepository = sakRepository;
+    }
+
+    MinSideBeskjedMorsArbeidTask() {
+        // CDI
+    }
+
+    @Override
+    public void doTask(ProsessTaskData data) {
+        var sak = sakRepository.hentFor(new Saksnummer(data.getSaksnummer()));
+        var saksnummer = sak.saksnummer();
+        if (sak.ytelse() != YtelseType.FORELDREPENGER) {
+            LOG.warn("Sendte ikke beskjed for sak {}. Kan ikke sende beskjed om mors arbeid for sak med ytelse {}", saksnummer, sak.ytelse());
+            return;
+        }
+        ForeldrepengerSak fpSak = (ForeldrepengerSak) sak;
+        if (!fpSak.brukerRolle().erFarEllerMedmor()) {
+            LOG.info("Sendte ikke beskjed for sak {}. Far/Medmor er ikke bruker", saksnummer);
+            return;
+        }
+        var annenPartAktørId = fpSak.annenPartAktørId();
+        if (annenPartAktørId == null) {
+            LOG.info("Sendte ikke beskjed for sak {}. Annen part mangler", saksnummer);
+            return;
+        }
+        if (!skalVarsleMor(fpSak)) {
+            LOG.info("Sendte ikke beskjed for sak {}. Ingen søknad der mor skal varsles", saksnummer);
+            return;
+        }
+        var eventId = UUID.nameUUIDFromBytes(saksnummer.value().getBytes(StandardCharsets.UTF_8)); //Samme saksummer gir samme uuid hver gang
```

Comment:
Hva tror dere om denne måten å gjøre det? Vi slipper å selv lagre hvilke saker som allerede har fått tilsendt varsel.
Vi gjør oss da litt avhengig av at minside ikke endrer på oppførsel rundt duplikate eventId'r, men jeg ser på det som lite sannsynlig. 
Dette sparer oss i hvertfall for en litt kode og kompleksitet

Reply:
Hva brukes ifm oppgavene rundt manglende vedlegg - eller tilbakekreving. Er de pr sak eller søknad?
Oppgaver må også lukkes av oss - så der må vi holde track på dem. Kan mindre om beskjeder/meldinger.
Dette er en grei måte å generere en id - men du kunne like godt brukt saksnummer ? Eller?
Nedsiden er at andre som får samme ideen kan møte på at man ikke får varslet bruker.

Reply:
Alternativet her er vel å piggy-backe OppgaveEntitet med en ny type med avsluttetTid = opprettetTid
Da kan du sjekke om det er sendt en melding med den oppgavetypen for saken.

Reply:
Enn så lenge så er det kanskje bare oppgave og beskjed som er tilgjengelig @dijjal  ?

Reply:
De beskjedene vi har i dag brukes lignende konsept om å sette en kjent UUID. Søknad mottatt beskjeden bruker eksternreferanse uuid fra journalføringhendelsen

Reply:
Du kan jo bruke bytes((saksnummer.value() + "beskjedMorsAktivitetArbeid").getBytes) - slik at andre beskjeder ikke blir hindret